app/services/adapters/jwe_webhook_destination.rb

```diff
@@ -29,6 +29,9 @@ def body_as_string(body)
     end
 
     def encrypted_body(body)
+      Rails.logger.debug '********************* DEBUG'
```

Comment:
Should remove the debug logs before merge (and possibly consider adding the key and url to the log filter?)